internal/collector/nginxossreceiver/internal/scraper/accesslog/testdata/test-access.log

```diff
@@ -1,8 +1,14 @@
 127.0.0.1 - - [16/Apr/2024:09:00:45 +0100] "GET /example HTTP/1.0" 200 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
+127.0.0.1 - - [16/Apr/2024:09:00:45 +0100] "GET /example HTTP/1.0" 200 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
+127.0.0.1 - - [16/Apr/2024:09:00:45 +0100] "GET /example HTTP/1.0" 200 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
 127.0.0.1 - - [16/Apr/2024:09:00:45 +0100] "GET /example HTTP/1.1" 203 28 "-" "PostmanRuntime/7.36.1" "-" "190" "235" "0.004" "-" "HTTP/1.1" "0.003""0.003" "28" "0.003" 
 127.0.0.1 - - [16/Apr/2024:09:03:02 +0100] "GET /example HTTP/1.0" 300 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
 127.0.0.1 - - [16/Apr/2024:09:03:02 +0100] "GET /example HTTP/1.1" 303 28 "-" "PostmanRuntime/7.36.1" "-" "190" "235" "0.002" "-" "HTTP/1.1" "0.000""0.003" "28" "0.003" 
 127.0.0.1 - - [16/Apr/2024:10:57:45 +0100] "GET /example HTTP/1.0" 400 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
+127.0.0.1 - - [16/Apr/2024:10:57:45 +0100] "GET /example HTTP/1.0" 400 28 "-" "PostmanRuntime/7.36.1" "-" "185" "222" "0.000" "-" "HTTP/1.0" "-""-" "-" "-" 
```

Comment:
What's the significance of adding duplicate lines here?

Reply:
I just wanted different count values for each response code. The test verifies that the number of response codes match what is in this file. So I wanted each number to be unique.  https://github.com/nginx/agent/blob/7ef0ac935dfb7532cd799e70d072d7dee5e3db91/internal/collector/nginxossreceiver/internal/scraper/accesslog/testdata/expected.yaml